app/javascript/components/recipes/edit/recipe-ingredients.vue

```diff
@@ -0,0 +1,100 @@
+          <add-ingredient-card
+            v-for="ingredient in this.ingredients"
+            :key="ingredient.id"
+            :name="ingredient.name"
+            :price="ingredient.price"
+          >
+            {{ ingredient.name }}
+          </add-ingredient-card>
+        </div>
+      </div>
+    </div>
+    <!-- ingredientes seleccionados -->
+    <div class="">
+      <div class="flex flex-col items-start w-96 h-full flex-none self-stretch flex-grow bg-gray-50">
+        <div class="flex items-start h-6 bg-gray-50 font-sans font-medium text-base text-black flex-none self-stretch flex-grow-0 mb-3">
+          {{ $t('msg.recipes.selectedIngredients') }}
+        </div>
+        <!-- {{ recipeIngredients.length }}
+        {{ recipeIngredients }} -->
+        <div
+          class="flex flex-col items-start w-96 h-96 flex-none flex-grow-0 bg-gray-200 overflow-scroll"
+          v-if="recipeIngredients.length > 0"
+        >
+          <selected-ingredient-card
+            v-for="ingredient in recipeIngredients"
+            :key="ingredient.id"
+            :name="ingredient.attributes.ingredient.name"
+            :price="ingredient.attributes.ingredient.price"
+          >
+            {{ ingredient.name }}
+          </selected-ingredient-card>
+        </div>
+        <div
+          class="flex items-start h-6 bg-gray-50 font-sans font-light text-base text-black flex-none self-stretch flex-grow-0 mb-3"
+          v-else
+        >
+          {{ $t('msg.recipes.noIngredients') }}
+        </div>
+        <div class="flex w-96 h-auto items-start justify-end py-4 px-2 bg-gray-200 border border-gray-300 box-border flex-none self-stretch flex-grow-0">
+          <div class="w-auto h-6 font-bold text-base text-black flex-none flex-grow-0 mx-2">
+            {{ $t('msg.recipes.recipePrice') }} ${{ recipePrice }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { getIngredients } from './../../../api/ingredients.js';
+import addIngredientCard from './add-ingredient-card.vue';
+import selectedIngredientCard from './selected-ingredient-card.vue';
+
+export default {
+  data() {
+    return {
+      ingredients: [],
+      error: '',
+    };
+  },
+  components: {
+    addIngredientCard,
+    selectedIngredientCard,
+  },
+  props: {
+    recipeIngredients: { type: Array, required: true },
+  },
+  async created() {
+    try {
+      const response = await getIngredients();
+      this.ingredients = response.data.data.map((element) => ({
+        id: element.id,
+        ...element.attributes,
+      }));
+      this.error = '';
+    } catch (error) {
+      this.error = error;
+    }
+  },
+  computed: {
+    recipePrice() {
+      // eslint-disable-next-line max-len
+      return this.recipeIngredients.map(ingredient => ingredient.attributes.ingredient.price).reduce((acc, curVal) => acc + curVal, 0);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 135. Maximum allowed is 120  vue/max-len

Reply:
Para que el mono no te moleste creo que tienes que ponerle en la misma linea `eslint-disable-line max-len`